kubernetes/config/kube_config.c

```diff
@@ -0,0 +1,309 @@
+#include "kube_config.h"
+#include "kube_config_yaml.h"
+#include <errno.h>
+#include <unistd.h>
+
+#define ENV_KUBECONFIGE "KUBECONFIG"
+#define ENV_HOME "HOME"
+#define KUBE_CONFIG_DEFAULT_LOCATION "%s/.kube/config"
+#define KUBE_CONFIG_TEMPFILE_NAME_TEMPLATE "/tmp/kubeconfig-XXXXXX"
+
+static int setBasePath(char **pBasePath, char *basePath)
+{
+    int rc = 0;
```

Comment:
is this necessary? modern c supports multi-return, so why not just:

```c
if (...) {
  return 0;
}
return -1;
```

Reply:
OK.